PrimeroAdministrationandConfigurationGuide.md

```diff
@@ -586,6 +588,16 @@ Once a Lookup is created, you can add to a field type and select whether you can
 
 To edit the properties of an existing Lookup, find the name and click _Edit_ in the options in the right column. From the _Show_ page, click the EDIT button at the top of the screen.
 
+### Managing Form Groups
+
+As you scroll through the list of lookups, you will notice that there is a lookup for the form groups for each type of record. Earlier, when we were creating a case (in the section **Creating and Editing a New Form**), we added our form to a new form group. When looking at the lookup labelled "Form Groups Cp Case," you will see that the form group you added appears here.
```

Comment:
We now have internal links working in our gitbook parser on the support hub. Maybe it is better to have these references as links?

> when we were creating a case (in the section `[**Creating and Editing a New Form**](#creating-and-editing-a-new-form))`, we added our form to a new form group. When looking at the lookup labelled "Form Groups Cp Case," you will see that the form group you added appears here.

Reply:
I tried making these work as internal links, but when I'd run `gitbook pdf ./`, the links in the resulting pdf file wouldn't work. If the support hub parser will be able to use these, I put the internal links back in, though.